dnase/callsnpsMerge.sh

```diff
@@ -72,10 +72,15 @@ date
 #NB: for hg38_full, many jobs will not generate coverage tracks
 covfiles=`cut -f1 ${sampleOutdir}/inputs.callsnps.${mappedgenome}.txt | xargs -I {} echo "${sampleOutdir}/${name}.${mappedgenome}.{}.coverage.starch"`
 covfiles=$(getFilesToMerge ${covfiles})
-starchcat ${covfiles} > ${sampleOutdir}/${name}.${mappedgenome}.coverage.starch
-rm -f ${covfiles}
 
-unstarch ${sampleOutdir}/${name}.${mappedgenome}.coverage.starch | cut -f1-3,5 > $TMPDIR/${name}.${mappedgenome}.coverage.bedGraph
+if [ "${covfiles}" = "" ]; then
```

Comment:
why would this be empty? doesn't your change above ensure inputs.callsnps has entries?